api/job/janitor_test.go

```diff
@@ -3,17 +3,43 @@ package job
 import (
 	"testing"
 	"time"
+
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestJanitor(t *testing.T) {
 	store := NewMemoryStore()
 	janitor := NewJanitor(store, time.Hour*24)
+	timeNow := time.Now()
+
+	jobs := []*models.Job{
+		{
+			JobID:   "delete",
+			Created: timeNow.Add(-24 * time.Hour),
+		},
+		{
+			JobID:   "keep",
+			Created: timeNow.Add(-12 * time.Hour),
+		},
+	}
 
-	// todo: put some jobs in the store
+	for _, job := range jobs {
```

Comment:
You should use a `mock_job.MockJobStore` here instead of the memory store. That way you can define the expected behavior more clearly:
```
mockJobStore.EXPECT().
    SelectAll().
    Return(jobs, nil)

mockJobStore.EXPECT().
    DeleteJob("delete").
    Return(nil)
```

and you won't need to do the select + assert behavior below

Reply:
I get an import cycle error when I try to use `mock_job.MockJobStore`